front-end-project/src/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+
+const baseUrl = "https://faizan-backend-project.onrender.com/api/";
```

Comment:
You could use axios.create here to create a variable that holds the baseUrl, then you wouldn't have to keep repeating {baseUrl} in your endpoints. Should be an example in the notes/docs. I'll leave it to you if you want to implement though!